### ISSUE TITLE
Typo in the main.css

### ISSUE THREAD

**Issue by SQLJames** [razonyang/hugo-theme-bootstrap#182]
Good morning, in investigating how to increase the contrast of some of the links in the website it looks like there is a typo in the main.css file for the colors of the navbar. After unminifying the css, and finding the element that corresponds to the coloring of these links we are using `--hbs-primary-tex` not `--hbs-primary-text` which does not seem to be defined. 

```
a {
    text-decoration: none;
    color: var(--hbs-primary-tex);
}
```

and 

```
.nav-link {
    color: var(--hbs-primary-tex);
}
```
https://web.dev/color-contrast/?utm_source=lighthouse&utm_medium=dotcom

Failing Elements
--
```

Search<input class="form-control rounded-pill" id="searchQuery" name="q" type="search" aria-label="Search">
Archives<a class="nav-link" href="/archives/">
Categories<a class="nav-link" href="/categories/">
Tags<a class="nav-link" href="/tags/">
Series<a class="nav-link" href="/series/">
Socials<a class="nav-link" id="navbarDropdown-dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
Settings<a class="nav-link" data-bs-toggle="offcanvas" href="#offcanvasSettings" role="button" aria-controls="offcanvasSettings">

```

**Comment by SQLJames** [comment]
https://github.com/razonyang/hugo-theme-bootstrap/pull/185

**Comment by razonyang** [comment]
Fixed by 95307965305be1a01a5a6671ac1de8fd3a0ad8ee.
Thanks for your contributions.
